src/Discord.Net.WebSocket/DiscordSocketClient.Events.cs

```diff
@@ -59,12 +59,12 @@ public partial class DiscordSocketClient
             remove { _messageReceivedEvent.Remove(value); }
         }
         private readonly AsyncEvent<Func<SocketMessage, Task>> _messageReceivedEvent = new AsyncEvent<Func<SocketMessage, Task>>();
-        public event Func<ulong, Optional<SocketMessage>, Task> MessageDeleted
+        public event Func<ulong, SocketChannel, Optional<SocketMessage>, Task> MessageDeleted
```

Comment:
As far as I can see only the UserIsTyping event uses ISocketMessageChannel. I'd not be opposed to that change tho.